stream_alert_cli/helpers.py

```diff
@@ -69,14 +69,16 @@ def run_command(runner_args, **kwargs):
 
 
 def continue_prompt(**kwargs):
-    """Continue prompt used before applying Terraform plans.
+    """Continue prompt to verify that a user wants to continue or not.
```

Comment:
This is still going to break other areas that call this, since they are not using keyword arguments. Can you update this function definition to accept one optional keyword arg like `def continue_prompt(message=''):` update all areas this is called?

Reply:
I updated all the callers....

Reply:
![screen shot 2017-09-19 at 2 44 25 pm](https://user-images.githubusercontent.com/11466941/30617011-10b88096-9d49-11e7-803a-8b29ab066c1d.png)

Reply:
LOL like that joke I just made